pkg/services/notifications/mailer.go

```diff
@@ -138,18 +147,25 @@ func (ns *NotificationService) buildEmailMessage(cmd *models.SendEmailCommand) (
 		return nil, models.ErrSmtpNotEnabled
 	}
 
-	var buffer bytes.Buffer
-	var err error
-
 	data := cmd.Data
 	if data == nil {
 		data = make(map[string]interface{}, 10)
 	}
 
 	setDefaultTemplateData(data, nil)
-	err = mailTemplates.ExecuteTemplate(&buffer, cmd.Template, data)
-	if err != nil {
-		return nil, err
+
+	body := make(map[string]string)
+	for _, contentType := range ns.Cfg.Smtp.ContentTypes {
+		var buffer bytes.Buffer
+		var err error
+
+		fileExtension := util.GetFileExtensionByContentType(contentType)
+		err = mailTemplates.ExecuteTemplate(&buffer, cmd.Template+fileExtension, data)
```

Comment:
There's no need to predeclare `err`.
```suggestion
		fileExtension := util.GetFileExtensionByContentType(contentType)
		var buffer bytes.Buffer
		err := mailTemplates.ExecuteTemplate(&buffer, cmd.Template+fileExtension, data)
```